lib/pcgs.gd

```diff
@@ -278,7 +278,7 @@ DeclareAttribute( "PcSeries", IsPcgs );
 ##  <Prop Name="IsPcgsElementaryAbelianSeries" Arg='pcgs'/>
 ##
 ##  <Description>
-##  returns <K>true</K> if the pcgs <A>pcgs</A> refines an elementary abelian
+##  returns <K>true</K> if the pcgs <A>pcgs</A> refines a elementary abelian
```

Comment:
Are you sure about that one?

My "feel of the language" (admittedly of a non-native speaker) makes me think "an elementary" is actually right, so does the rule on it I learned on it (when pronounced, the word starts with a sounding vowel).
 "Google fight" seems to agree: 15 million hits for the former and tons of books with "an elementary introduction" in the title; only 400k hits for "a elementary", and many of them seem to be for "big A elementary".

Or perhaps this is an American vs. British english thing?